### ISSUE TITLE
Request for Information - Add images to local CVAT instance to an existing task or a new task,

### ISSUE THREAD

**Issue by sammed-ai** [openvinotoolkit/cvat#3405]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

A user wishes to upload a batch of images to my local CVAT platform. I am looking for an API endpoint which will allow the user to upload images to an existing task with an existing Job. Is this possible? 
Or do I have to create a new task? But creation of new task API only creates a new task without anyway to create a job within the task and add images. I am stuck in a loop. 

Are there a set of API endpoints that I can use so that the user can upload image via the endpoints to my CVAT platform? I do not care if it leads to creation of a new task or a new job. I need to the images to just upload to my CVAT platform. 

As far as I know using the UI, once a task is created I cannot create a job within the task and once a task and job are crated I cannot add images to the existing job. So, how do I create a new task, a new Job and add images/videos to the job in one go?

Since this is a request for information the following questions do not require any answers. 

### Steps to Reproduce (for bugs)

1.
1.
1.
1.

### Your Environment

- Git hash commit (`git log -1`):
- Docker version `docker version` (e.g. Docker 17.0.05):
- Are you using Docker Swarm or Kubernetes?
- Operating System and version (e.g. Linux, Windows, MacOS):
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

**Comment by bsekachev** [comment]
>  user to upload images to an existing task with an existing Job. Is this possible?

No, it is not possible. Please, refer to the original issue #95. 


You can add images/video to a created task only once. Depending on this request a set of jobs is created within the task. 
Refer to API specification (localhost:8080/api/swagger) for details
![Screenshot from 2021-07-19 11-37-47](https://user-images.githubusercontent.com/40690378/126129868-9b7603f3-5a17-4092-9183-4ec6eae09115.png)

**Comment by bsekachev** [comment]
I will close the issue, because there is not a response from your side. Do not hesitate to reopen it if you still have questions
